lib/contentTypeParser.js

```diff
@@ -216,6 +275,12 @@ function Parser (asString, asBuffer, bodyLimit, fn) {
   this.asBuffer = asBuffer
   this.bodyLimit = bodyLimit
   this.fn = fn
+
+  // Check for deprecation syntax
+  if (fn.length === (fn.constructor.name === 'AsyncFunction' ? 1 : 2)) {
+    emitWarning('FSTDEP003')
```

Comment:
Is this warn emitted for every parsing?
Does it is too much?

Reply:
Well, this is only done during addContentTypeParser. And anyway in `lib/warnings.js` there are checks to only emit a warning once.